cli/src/action/circuit/mod.rs

```diff
@@ -161,6 +161,11 @@ impl Action for CircuitProposeAction {
         }
 
         if let Some(comments) = args.value_of("display_name") {
+            if args.is_present("compat-0.4") {
```

Comment:
I think we should go with --compat=0.4 (so that the list of arguments doesn't grow as we do releases).

Reply:
ah okay

Reply:
updated